larray/tests/data/examples/fertility.csv

```diff
@@ -0,0 +1,7 @@
+geo,gender\time,2013,2014,2015
+BE,M,64371,64173,62561
```

Comment:
These seem to be birth numbers, not fertility (which I *assume* are usually given as rates).

Reply:
The dataset comes from ``Database by themes > Population and social conditions > Demography and migration (demo) > Fertility``